rancher/dashboard
# Added blue color only to creation action

### Summary
Fixes #6412

In the namespace selection dropdown, only "Create a New Namespace" should be blue

### How to test
Go to : https://192.168.43.142:8005/c/local/explorer/apps.deployment
Click on the `Create` button => Namespace dropdown should have **"Create a New Namespace"** in blue color and rest of the options should be in standard text color.
